core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -52,20 +68,23 @@ default void onSuccess(
    * @param executionProfile the execution profile of this request.
    * @param node the node that returned the successful response.
    * @param requestLogPrefix the dedicated log prefix for this request
+   * @param executionInfo the execution info containing the results of this request
    */
   default void onSuccess(
       @NonNull Request request,
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
-      @NonNull String requestLogPrefix) {
-    // If client doesn't override onSuccess with requestLogPrefix delegate call to the old method
-    onSuccess(request, latencyNanos, executionProfile, node);
+      @NonNull String requestLogPrefix,
```

Comment:
Small request: would it be possible to keep the log prefix as the _last_ parameter? This has been kind of a tacit convention so far.

Reply:
Refactored to make requestLogPrefix the last arg in all the methods.